aries-site/src/examples/templates/dashboards/DashboardExample.js

```diff
@@ -1,6 +1,237 @@
-import React from 'react';
-import { DashboardTiles } from '.';
-import { AppHeaderExample, PageHeaderExample, ScreenContainer } from '..';
+import React, { useContext } from 'react';
+import {
+  Avatar,
+  Header,
+  Heading,
+  Button,
+  Box,
+  Grid,
+  Menu,
+  ResponsiveContext,
+  Footer,
+  Text,
+} from 'grommet';
+import PropTypes from 'prop-types';
+
+import {
+  Apps,
+  Chat,
+  Clock,
+  Hpe,
+  Terminal,
+  StatusUnknown,
+  Wifi,
+  System,
+  User,
+  Tasks,
+  Location,
+  ShieldSecurity,
+} from 'grommet-icons';
+
+const data = [
+  {
+    color: 'blue',
+    icon: <Wifi />,
+    title: 'Remote Access',
+    subTitle: 'Lights out Management (LOM)',
+    message: 'Connected',
+  },
+  {
+    color: 'green',
+    icon: <System />,
+    title: 'System',
+    subTitle: 'Sub-system and Devices',
+    message: 'Composable System',
+  },
+  {
+    color: 'red',
+    icon: <User />,
+    title: 'User Sessions',
+    subTitle: 'User Access on Server',
+    message: '4 active sessions',
+  },
+  {
+    color: 'purple',
+    icon: <Tasks />,
+    title: 'Logs',
+    subTitle: 'Events, Integration, and Status',
+    message: '204,353',
+  },
+  {
+    color: 'orange',
+    icon: <Location />,
+    title: 'Beacons',
+    subTitle: 'Uniqe identification light',
+    message: '24 beacons connected',
+  },
+  {
+    color: 'teal',
+    icon: <ShieldSecurity />,
+    title: 'Security',
+    subTitle: 'Trusted Platform Module',
+    message: 'No Module Connected',
+  },
+];
+
+const DashboardTiles = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Grid
+      gap="medium"
+      columns={size !== 'small' ? 'small' : 'auto'}
+      margin={size === 'small' ? { bottom: 'xlarge' } : undefined}
+    >
+      {data.map(value => (
+        <Box
+          round="small"
+          flex
+          fill
+          background={value.color}
+          key={`Tile ${value.title}`}
+          alignContent="center"
+          onClick={() => {}}
+        >
+          <Box
+            pad={{ horizontal: 'medium', vertical: 'small' }}
+            gap="medium"
+            justify="between"
+            size="small"
+            direction="column"
+            align="start"
+          >
+            {value.icon}
+            <Box gap="medium">
+              <Box>
+                <Text size="medium" weight="bold">
+                  {value.title}
+                </Text>
+                <Text size="medium">{value.subTitle}</Text>
+              </Box>
+            </Box>
+          </Box>
+          <Box flex />
```

Comment:
Is there any way to get the same spacing without the empty box?

Reply:
maybe some kind of justify between somewhere in a parent box? not sure, but just first thing that came to mind